build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -238,8 +238,11 @@ service ActionCache {
 //   the uploaded data once uncompressed, and MUST return an
 //   `INVALID_ARGUMENT` error in the case of mismatch.
 //
-// Note that when writing compressed blobs, the `WriteRequest.write_offset`
-// refers to the offset in the uncompressed form of the blob.
+// Note that when writing compressed blobs, the `WriteRequest.write_offset` in
+// the initial request refers to the offset in the uncompressed form of the
+// blob. In subsequent requests, `WriteRequest.write_offset` MUST be the sum of
+// the previous `WriteRequest.write_offset` and the size of the compressed chunk
```

Comment:
I think nit:

`MUST be the sum of the first request's 'WriteRequest.write_offset' and the total size of all the compressed chunks in the previous requests`. 

And maybe `chunks` -> `data bundles`? Since that's the word used in the ByteStream spec.

Reply:
Done.